### DISCUSSION TITLE
Hit-highlight specific managed properties in custom template

### DISCUSSION THREAD

**Discussion by jojocode77** [microsoft-search/pnp-modern-search#3784] [Q&A]
I'm sure it'll be really obvious, but I can't seem to figure this one out.

I have a custom display template to display data from a list that lays out a number of different managed properties via eg {{slot item @root.slots.SlotName}} in a nice layout (apologies for the obfuscation, but each black box is a different managed property)

I would like to highlight the matching search term in each area, for example, if someone searches "share plans" I'd like to highlight that everywhere it appears. In this case, it's just in that 3rd column property, but it might be in the first or middle as well / instead.

I've added the properties to the HitHighlightedProperties section, and then I'm stuck. I've tried calling {{getSummary "SlotName"}} but that doesn't do the job. I can see the hit highlight in the HitHighlightedSummary that's returned, but I don't know how to get just the bit I need to display in the relevant part of my template.

Is it possible at all?

Thank you!
![image](https://github.com/microsoft-search/pnp-modern-search/assets/23478322/408fa6d0-eb86-4f91-b919-513d47ee9472)

**Comment by wobba** [comment]
You have to pull content from the `HitHighlightedProperties` property coming back. This is in XML and not sure if getSummary will work on that one or not. Switch to debug layout and you see the format of this property.
